front-end/src/components/UserInformationScreen.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import Divider from '@mui/material/Divider';
+import Typography from '@mui/material/Typography';
+
+
+const courseList =[
```

Comment:
Try separating out this test data into its own file under the "test-data" folder, then import the test data into this component.